definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/homer:0.1"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+                set -o pipefail
+                set -o errexit
+
+                name="homer_tag_directory"
+                bam=$1
+                reference_fasta=$2
+
+                export PATH=$PATH:/opt/homer/bin
```

Comment:
If this is necessary for the proper operation of `homer`, it should be part of the Docker image.  In fact it already appears to be:
https://github.com/genome/docker-homer/blob/33adf12e1d5e1a49e8e12f72adb61352fa02d214/Dockerfile#L31

While it's harmless to add it a second time, it shouldn't be needed.

Reply:
The PATH sometimes doesn't get preserved in our interactive docker containers, which is why we have gotten into the habit of exporting the path in any script that uses homer.  If you can remove it and things still run properly, though, Sai, then sure, drop that line.

Reply:
On our cluster, you'd have to disable the envrionment preservation when running interactively.  This is included in the CWL pipeline runner:
https://github.com/genome/genome/blob/master/lib/perl/Genome/Model/CwlPipeline/Command/Run.pm#L235

Reply:
I removed the export PATH line and it worked so I guess that solves the issue. I just had to add /opt/homer/bin in front of makeTagDirectory